integr8ly/application-monitoring-operator
# Prepare release 1.8.1 - Re-adding ClusterRole for Blackbox ServiceAccount that was removed

# Description
The ClusterRole for the blackbox-service-account (originally added in version [1.7.0](https://github.com/integr8ly/application-monitoring-operator/pull/170)) was removed when AMO was bumped to [1.8.0](https://github.com/integr8ly/application-monitoring-operator/pull/167). This PR adds the ClusterRole back into the CSV.

JIRA: https://issues.redhat.com/browse/MGDAPI-2282

## Verification
1. Create a new OSD cluster via [OpenShift](https://qaprodauth.cloud.redhat.com/openshift/) and target it once it has finished provisioning.
2. Pull the changes from this PR and apply the new ClusterRole and ClusterRoleBinding to the remote cluster:
`oc apply -f application-monitoring-operator/deploy/cluster-roles/blackbox-exporter-clusterrole.yaml`
`oc apply -f application-monitoring-operator/deploy/cluster-roles/blackbox-exporter-clusterrole_binding.yaml`
2. Replace [this line](https://github.com/integr8ly/integreatly-operator/blob/master/manifests/integreatly-monitoring/1.8.0/application-monitoring-operator.v1.8.0.clusterserviceversion.yaml#L252) in integreatly-operator to point the image stored in my Quay [repository](https://quay.io/repository/ckyrillo/application-monitoring-operator):
`image: quay.io/ckyrillo/application-monitoring-operator:v1.8.1`
3. Install RHOAM from the command line to the remote cluster using the PR from Step 3.
4. Once the installation is complete, verify that the "grafana-ui UP/DOWN Status" panel in the Endpoint Dashboard from the redhat-rhoam-middleware-monitoring-operator grafana route correctly displays that the grafana service is UP.

Screenshot of the dashboard without the bugfix:
![MGDAPI-2282-bug](https://user-images.githubusercontent.com/80267718/129981866-50ce33ff-d87f-4593-8875-edb5ac8e88da.png)

Screenshot of the dashboard with the bugfix:
![MGDAPI-2282-bugfix](https://user-images.githubusercontent.com/80267718/129980186-8493c3f8-b11f-41d7-9abb-3a42828fe232.png)


## Type of change
